pop-cli-for-appchains/SUMMARY.md

```diff
@@ -18,12 +18,9 @@
 * [Launch a Chain](guides/launch-a-chain/README.md)
   * [Launch a Chain in Development](guides/launch-a-chain/running-your-parachain.md)
   * [Launch a Chain to Paseo](guides/launch-a-chain/launch-a-chain-to-paseo.md)
+    * [Spinning up Paseo](guides/spinning-up-paseo.md)
```

Comment:
Maybe let's just get rid of the "spinning up" wording and let's say Launch instead.

Reply:
Launch Paseo sounds better option: https://github.com/r0gue-io/pop-docs/pull/29/commits/ec98289cd23467413f56a30d4a760068ffb4420e